packages/driver/src/cy/commands/origin/index.ts

```diff
@@ -170,21 +170,21 @@ export default (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy, state: State
         }
 
         // fired once the spec bridge is set up and ready to receive messages
-        communicator.once('bridge:ready', async (_data, specBridgeOriginPolicy) => {
-          if (specBridgeOriginPolicy === originPolicy) {
+        communicator.once('bridge:ready', async (_data, specBridgeOrigin) => {
```

Comment:
this should be specBridgeSuperDomainOrigin, or we should convert it into a superDomainOrigin before checking it agains the superDomainOrigin

Reply:
I think this should be `specBridgeSuperDomainOrigin`. I need to take a pass and rename a few of function params as well as state keys, like `latestActiveOrigin` and `currentActiveOrigin`

Reply:
should be up to date in 1ee8e686a41d0bb4e723c6faf4df4a301636edea